src/app/components/home-feed/home-feed.component.scss

```diff
@@ -1,17 +1,25 @@
 main {
-  padding: 80px 0px;
-  display: grid;
-  grid-template-columns: auto auto auto;
+    padding: 80px 50px;
+    width: 600px;
 }
 
-@media screen and (max-width:400px) {
-  main {
-    grid-template-columns: auto auto;
-  }
+.image-size {
+    width: 30px;
+    height: 30px;
+    object-fit: cover;
 }
 
-@media screen and (max-width:300px) {
-  main {
-    grid-template-columns: auto;
-  }
+.timestamp {
```

Comment:
you can use bootstraps `.text-secondary` for this

Reply:
Done!

Reply:
Could you remove .timestamp css class please

Reply:
Done !